cuda/matrix/dense_kernels.cu

```diff
@@ -120,9 +121,9 @@ __global__ __launch_bounds__(block_size) void scale(
     const ValueType *__restrict__ alpha, ValueType *__restrict__ x,
     size_type stride_x)
 {
-    constexpr auto warps_per_block = block_size / cuda_config::warp_size;
+    constexpr auto warps_per_block = block_size / config::warp_size;
```

Comment:
We currently have many dense kernels duplicated in `common` and `cuda`. Can we easily reuse the code from `common` here?

Reply:
I can see that's part of the other pull request. So just ignore my comment :)